simapp/simd/cmd/root.go

```diff
@@ -157,14 +157,8 @@ func initRootCmd(rootCmd *cobra.Command, encodingConfig params.EncodingConfig) {
 	cfg := sdk.GetConfig()
 	cfg.Seal()
 
-	creator := func(logger log.Logger, db dbm.DB, options servertypes.AppOptions) (servertypes.RootMultiStoreProvider, error) {
-		pruningOptions, err := server.GetPruningOptionsFromFlags(options)
-		if err != nil {
-			return nil, err
-		}
-
-		app := simapp.NewSimApp(logger, db, nil, true, encodingConfig, nil, baseapp.SetPruning(pruningOptions))
-		return app, nil
+	creator := appCreator{
```

Comment:
we can share the same `appCreator` as the below one I think?

Reply:
yeah, sure. 
lifted the below appCreator